Makefile

```diff
@@ -53,6 +53,15 @@ copy-resources:
 	cp training/src/main/resources/com/google/myanmartools/compatibility.tsv clients/ruby/lib/myanmar-tools/resources
 	cp training/src/main/resources/com/google/myanmartools/compatibility.tsv clients/php/resources
 
+	cp genconvert/input/udhr_mya.txt clients/js/resources
+	cp genconvert/input/z_3022.txt clients/js/resources
+	cp genconvert/output/udhr_mya_zawygyi.txt clients/js/resources
+	cp genconvert/output/u_3022.txt clients/js/resources
```

Comment:
Can you make better names for these files?  Suggestions:

- `udhr_unicode_src.txt`
- `udhr_zawgyi_out.txt`
- `mmgov_zawgyi_src.txt`
- `mmgov_unicode_out.txt`

Reply:
Updated.